tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -991,3 +991,76 @@ def process(self, image, annotation_meta=None):
         })
 
         return image
+
+class PersonVehicleDetectionPatch(Preprocessor):
```

Comment:
I see that it makes the same things like face_detection_patch preprocessor. Can we reuse the same preprocessor for both face and person-vehicle cases, just rename it to DetectionPatch?